src/autopas/containers/verletClusterCells/VerletClusterCells.h

```diff
@@ -170,15 +170,15 @@ class VerletClusterCells : public ParticleContainer<FullParticleCell<Particle>>,
     _isValid = false;
     for (size_t i = 0; i < this->_cells.size(); ++i) {
       for (size_t j = 0; j < _dummyStarts[i];) {
-        if (not this->_cells[i]._particles[j].isOwned()) {
+        if (not this->_cells[i].at(j).isOwned()) {
```

Comment:
why use `at()` over `[]` here? Access without boundary check should be fine here.

This goes for all similar changes in this class.

Reply:
I changed it everywhere except for here: 
std::swap(this->_cells[i][j], this->_cells[i].at(_dummyStarts[i])); 

because I think _cells[i][_dummyStarts[i]]); is quite a bit harder to read

Reply:
the difference is not only in readability but `.at()` makes a check if the index is actually valid at runtime, thus adding overhead.

Reply:
Oh okay I'll remove it then